lib/matplotlib/backends/backend_cairo.py

```diff
@@ -286,10 +286,18 @@ def get_canvas_width_height(self):
     def get_text_width_height_descent(self, s, prop, ismath):
         # docstring inherited
 
+        if ismath == 'TeX':
+            # todo: handle props
+            texmanager = self._text2path.get_texmanager()
+            fontsize = prop.get_size_in_points()
+            w, h, d = texmanager.get_text_width_height_descent(
+                s, fontsize, renderer=self)
+            return w, h, d
+
+        dpi = self.points_to_pixels(72)
         if ismath:
-            width, height, descent, fonts, used_characters = \
```

Comment:
this part should be left as is, i.e. in the mathtext case the parsing should be done using MathtextBackendCairo (`self.mathtext_parser`) instead of MathtextBackendPath (`self._text2path.mathtext_parser`) as we're going to use the former for rendering too (so the extents should also be computed using the former).

Reply:
Right, but do we still need the `self.dpi` -> `self.points_to_pixels(72)` change then?

Reply:
Looking at its code, I think we can drop that from both Cairo and Agg.